docs/user_guide/config.md

```diff
@@ -187,286 +187,293 @@ To add one or more local nodes to the list of nodes known to your f-API, simply
-
-This file adds:
-
-- [nginx-proxy](https://github.com/nginx-proxy/nginx-proxy) to run NGINX in a container, and automatically generate/refresh NGINX reverse proxy configurations when Neurobagel service containers are started
-- [acme-companion](https://github.com/nginx-proxy/acme-companion) to automatically create and renew SSL certificates for NGINX proxied service containers
-
-
-**Example `docker-compose.yml` with Reverse Proxy**
-
-To use this file:
+Below is an example implementation of a reverse proxy, using a custom Docker Compose file that builds on [`docker-compose.yml`](https://github.com/neurobagel/recipes/blob/main/docker-compose.yml) in the default Neurobagel deployment recipe: 
 
 1. If you haven't already, follow the [steps](getting_started.md#the-neurobagel-node-deployment-recipe) to clone and minimally configure the services in the [Neurobagel deployment recipe](https://github.com/neurobagel/recipes).
-2. Replace the default `docker-compose.yml` in the `recipes` directory with the appropriate file from below.
-3. Select the reverse proxy recipe you prefer:
-
-=== "NGINX"
-    ??? abstract "NGINX Reverse Proxy Configuration"
-        **To use this file:**
-
-        - Open the file and edit the values of `VIRTUAL_HOST` and `LETSENCRYPT_HOST` to the domains your proxied services will use.
-
-        - This assumes you have already registered your domain(s) with a DNS provider and configured the DNS settings to resolve correctly to your host machine.
-
-        **`docker-compose.yml`:**
-        ```yaml
-         services:
-          api:
-            image: "neurobagel/api:${NB_NAPI_TAG:-latest}"
-            profiles:
-              - "local_node"
-              - "full_stack"
-            ports:
-              - "${NB_NAPI_PORT_HOST:-8000}:8000"
-            environment:
-              NB_GRAPH_USERNAME: ${NB_GRAPH_USERNAME}
-              NB_GRAPH_ADDRESS: graph
-              NB_GRAPH_PORT: 7200
-              NB_GRAPH_DB: ${NB_GRAPH_DB:-repositories/my_db}
-              NB_RETURN_AGG: ${NB_RETURN_AGG:-true}
-              NB_API_PORT: 8000
-              NB_API_ALLOWED_ORIGINS: ${NB_NAPI_ALLOWED_ORIGINS}
-              NB_ENABLE_AUTH: ${NB_ENABLE_AUTH:-false}
-              NB_QUERY_CLIENT_ID: ${NB_QUERY_CLIENT_ID}
-              VIRTUAL_HOST: myservice1.myinstitute.org 
-              LETSENCRYPT_HOST: myservice1.myinstitute.org 
-              VIRTUAL_PORT: 8000
-            volumes:
-              - "./scripts/api_entrypoint.sh:/usr/src/api_entrypoint.sh"
-            entrypoint:
-              - "/usr/src/api_entrypoint.sh"
-            secrets:
-              - db_user_password
+2. Replace the default `docker-compose.yml` in the `recipes` directory with the appropriate file(s) below which contain an example reverse proxy configuration, based on the reverse proxy server you are using:
```

Comment:
```suggestion
2. Replace the default `docker-compose.yml` in the `recipes` directory with the appropriate file(s) below which contain an example reverse proxy configuration, based on the reverse proxy server want to use:
```
These recipes are not really compatible with an existing reverse proxy setup - in fact that's what got Thomas initially confused